public/index.html

```diff
@@ -1,37 +1,290 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="ru">
+<head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/Spotify.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+
+    <title> Spotify </title>
+    <link rel="stylesheet" href="./styles.css" />
```

Comment:
стоит использовать путь вида
```suggestion
    <link rel="stylesheet" href="%PUBLIC_URL%/styles.css" />
```

Картинки точно также стоит подключать

После npm run build подставятся корректны пути